README.md

```diff
@@ -1,26 +1,19 @@
-<h1 align="center">OUDS Android</h1>
+# OUDS Android
 
-<p align="center">
-  OUDS Android provides Orange Android components to developers and a demo application.
-  <br>
-    <a href="https://android.unified-design-system.orange.com/"><strong>Visit OUDS Android</strong></a>
-  <br>
-  <br>
-  <a href="https://github.com/Orange-OpenSource/ouds-android/issues/new?assignees=paulinea&labels=%F0%9F%90%9E+bug%2C%F0%9F%94%8D+triage&template=bug_report.yml&title=[Bug]%3A+Bug+Summary">Report bug</a>
-  ·
-  <a href="https://github.com/Orange-OpenSource/ouds-android/issues/new?assignees=paulinea&labels=feature%2C%F0%9F%94%8D%20triage&template=feature_request.yml&title=[feature]%3A+">Request feature</a>
-</p>
+[![minSdkVersion](https://img.shields.io/badge/minSdkVersion-21-yellowgreen?logo=android&logoColor=white)](https://apilevels.com)
+[![License](https://img.shields.io/github/license/Orange-OpenSource/ouds-android)](LICENSE)
+[![Documentation](https://img.shields.io/badge/documentation-7F52FF?logo=kotlin&logoColor=white)](https://android.unified-design-system.orange.com/)
```

Comment:
For information, why did you choose 7F52FF as the badge color?

Reply:
This is the Kotlin color [here](https://simpleicons.org/?q=kotlin)